mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_notification_config_test.go

```diff
@@ -0,0 +1,139 @@
+package securitycenterv2_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/securitycenter/v1"
+)
+
+func TestAccSecurityCenterOrganizationNotificationConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"topic_name":    acctest.RandString(t, 10),
+		"config_id":     acctest.RandString(t, 10),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckSecurityCenterOrganizationNotificationConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterOrganizationNotificationConfig_basic(context),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
You shouldn't need `Check`s here and below because the value checking is covered by the ImportStateVerify

Reply:
addressed